CHANGELOG.md

```diff
@@ -203,7 +203,7 @@ Generated by [`auto-changelog`](https://github.com/CookPete/auto-changelog).
 
 ### Merged
 
-- feat(env): add command to list/get/set/import environment variables  [`#1162`](https://github.com/netlify/cli/pull/1162)
+- feat(env): add command to list/get/set/import environment variables [`#1162`](https://github.com/netlify/cli/pull/1162)
```

Comment:
Can we ignore the this file and the `README.md` file for now?
Some parts of those files are autogenerated (we should fix this in the generation code).